course/src/main/java/in/testpress/course/ui/PdfViewerActivity.kt

```diff
@@ -0,0 +1,47 @@
+package `in`.testpress.course.ui
+
+import `in`.testpress.course.R
+import android.os.AsyncTask
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.layout_pdf_viewer.*
+import java.io.BufferedInputStream
+import java.io.InputStream
+import java.lang.Exception
+import java.net.HttpURLConnection
+import java.net.URL
+
+class PdfViewerActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.layout_pdf_viewer)
+        val file = intent.getStringExtra("fileUrl") ?: ""
```

Comment:
Class name is PDFViewActivity but why is this file Url. Change it as pdfUrl variable name also should be same